tests/snmpsim/hpvc.snmprec

```diff
@@ -1 +1,2 @@
 1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.11.5.7.5.1
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.11.5.7.5.4
```

Comment:
You need to put this in a new snmprec file. It also needs to include the sysDescr.